scripts/utils/exec.ts

```diff
@@ -13,28 +15,22 @@ async function exec(txData, totalValue, addresses) {
   const { proposerAddress, voterAddress, governorAlphaAddress } = addresses;
 
   // Impersonate the proposer and voter with sufficient TRIBE for execution
-  await hre.network.provider.request({
-    method: 'hardhat_impersonateAccount',
-    params: [proposerAddress]
-  });
-
-  await hre.network.provider.request({
-    method: 'hardhat_impersonateAccount',
-    params: [voterAddress]
-  });
+  await hre.network.provider.request({ method: 'hardhat_impersonateAccount', params: [proposerAddress] });
+  await hre.network.provider.request({ method: 'hardhat_impersonateAccount', params: [voterAddress] });
 
   // Submit proposal to the DAO
   if (txData) {
     console.log('Submitting Proposal');
-    await web3.eth.sendTransaction({
-      from: proposerAddress,
+    await (
+      await ethers.getSigner(proposerAddress)
+    ).sendTransaction({
       to: governorAlphaAddress,
       data: txData,
-      gas: 6000000
+      gasLimit: 6000000
     });
   }
 
-  const governor = await GovernorAlpha.at(governorAlphaAddress);
+  const governor = await ethers.getContractAt('GovernorAlpha.', governorAlphaAddress);
```

Comment:
is there supposed to be a period in here?

Reply:
No - fixed.